admin_manual/configuration/server/occ_command.rst

```diff
@@ -1893,8 +1896,50 @@ To delete a user, you use the ``user:delete`` command, as in the example below:
 ::
 
  sudo -u www-data php occ user:delete fred
-   
-   
+
+Expiring a User’s Password
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To expire a user’s password at a given time, use the ``user:expire-password`` command, as in the example below:
+
+::
+
+  # Expire the user "frank"’s password 24 hours prior
+  occ user:expire-password frank
+
+  # Expire the user "frank"’s password in 2 days
+  occ user:expire-password frank '+2 days'
+
+  # Expire the user "frank"’s password on the 15th of August 2005, at 15:52:01 in the local timezone
+  occ user:expire-password frank '2005-08-15T15:52:01+00:00'
+
+  # Expire the user "frank"’s password on the 15th of August 2005, at 15:52:01 UTC
+  occ user:expire-password frank '15-Aug-05 15:52:01 UTC'
+
+
+The command accepts expiry dates in any of `PHP’s supported date and time formats`_, and defaults to the previous day if an expiry date is not supplied.
+When the command runs, you will see a confirmation of when the password is set to expire, printed to the console, as in the example output below.
+
+::
+
+  The password for frank is set to expire on 2018-07-12 13:15:28 UTC.
+
```

Comment:
Pease move that to the top of the topic,
The "please be aware" part can be imho post the examples as it is now